src/service/repo/anonymousMessageRepo.ts

```diff
@@ -18,6 +20,7 @@ interface AnonymousMessageRepoTypes {
   getAllAnonymousPostNotifications: () => Promise<AnonymousPostNotification[]>;
   getSingleAnonymousPostNotifications: (activityId: string) => Promise<AnonymousPostNotification>;
   setChannelAsRead: (channelId: string) => Promise<boolean>;
+  deleteMessage: (messageId: string) => Promise<any>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return type of `deleteMessage` function is `Promise<any>`. It's generally a good practice to avoid using `any` in TypeScript as it defeats the purpose of static typing. If possible, replace `any` with a more specific type.



<!-- This is an auto-generated comment by OSS CodeRabbit -->